components/Header.js

```diff
@@ -1,45 +1,48 @@
 import styled from "styled-components";
 import {useGlobalState, setGlobalState} from "../state/index";
 import Bookmark from "../public/images/bookmark.svg";
+import BookmarkBlack from "../public/images/Star_black.svg";
 import Image from "next/image";
-import {signOut, getSession} from "next-auth/react";
 import SignOut from "../public/images/signout.svg";
+import {useRouter} from "next/router";
 
 export default function Header() {
   const [metamaskAddress] = useGlobalState("metamaskAddress");
+  const [chain] = useGlobalState("chainId");
+  const [popupState] = useGlobalState("openPopup");
+  const [Connecting] = useGlobalState("isConnecting");
+  const path = useRouter().asPath;
+  const [isConnected] = useGlobalState("isConnected");
 
   function openPopup() {
-    setGlobalState("openPopup", true);
+    if (path === "/home") {
+      setGlobalState("openPopup", true);
+    }
   }
 
   return (
     <>
```

Comment:
You don't need a fraction `<>` element at this point. Your `<StyledHeader>` is already your top level element.

Reply:
done!